x/ownership/handler.go

```diff
@@ -0,0 +1,17 @@
+package ownership
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/mesg-foundation/engine/x/ownership/internal/types"
+)
+
+// NewHandler creates an sdk.Handler for all the instance type messages
+func NewHandler(k Keeper) sdk.Handler {
+	return func(ctx sdk.Context, msg sdk.Msg) (*sdk.Result, error) {
+		errMsg := fmt.Sprintf("unrecognized %s message type: %T", types.ModuleName, msg)
```

Comment:
The module should also use its alias.
The alias system is made to allow customisation of the module. So the main package `x/ownership` should use them, i think.
```suggestion
		errMsg := fmt.Sprintf("unrecognized %s message type: %T", ModuleName, msg)
```

Reply:
It's taken from the template. I think the alias is for other packages, because the module can use `internal/` safely.